server/src/routes/router.ts

```diff
@@ -0,0 +1,10 @@
+import { Router } from 'express'
+import userController from '../controllers/userController'
+import { Schemas, ValidateSchema } from '../middleware/ValidateSchema'
+
+const router = Router()
+
+router.post('/login', ValidateSchema(Schemas.user.login), userController.login)
+router.post('/register-user', ValidateSchema(Schemas.user.register), userController.register)
```

Comment:
i suggest naming the route just register, what else are you going to register besides users ?